infra/build/functions/build_and_run_coverage.py

```diff
@@ -223,6 +238,73 @@ def get_build_steps(  # pylint: disable=too-many-locals, too-many-arguments
       build_lib.http_upload_step(latest_report_info_body,
                                  latest_report_info_url,
                                  LATEST_REPORT_INFO_CONTENT_TYPE))
+
+  if introspector_enabled:
+    coverage_url = bucket.html_report_url
+    build_steps.extend(
+        get_fuzz_introspector_steps(project, base_images_project, config,
+                                    coverage_url))
+  return build_steps
+
+
+def get_fuzz_introspector_steps(project, base_images_project, config,
+                                coverage_url):
+  """Return build steps of fuzz introspector for project"""
+  build_steps = []
+  build = build_project.Build(FUZZING_ENGINE, 'introspector', ARCHITECTURE)
+  env = build_project.get_env(project.fuzzing_language, build)
+
+  report_date = build_project.get_datetime_now().strftime('%Y%m%d')
+  bucket = IntrospectorBucket(project.name, report_date, PLATFORM,
+                              config.testing)
+
+  build_steps.append({
+      'name':
+          build_project.get_runner_image_name(base_images_project,
+                                              config.test_image_suffix),
+      'args': [
+          'bash', '-c',
+          ('sed -i s/base-builder/base-builder:introspector/g '
```

Comment:
So what you can do to avoid this is the following
run the command:
`docker tag gcr.io/oss-fuzz-base/base-builder:introspector gcr.io/oss-fuzz-base/base-builder:latest`
and then you won't need sed.

Note that your technique and mine breaks if someone were to change the FROM line.
This happens sometimes when someone wants to use an older version of clang.

Reply:
+1. Do this instead of sed.

Reply:
Done!